shared/hashutil/hash.go

```diff
@@ -0,0 +1,14 @@
+package hashutil
+
+import (
+	"golang.org/x/crypto/blake2b"
+)
+
+// Hash defines a function that returns the
```

Comment:
can you add this [link](https://github.com/ethereum/eth2.0-specs/blob/master/specs/beacon-chain.md#appendix-a---hash-function) in the comments? thanks